src/components/AddContactForm.js

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  MESSAGE_ERROS,
+  TITLE,
+  PLACEHOLDERS,
+  LABELS,
+  GET_REGEX,
+  MAX_LENGTH_NAME
+} from '../constants/constants';
+import { validateInput } from '../helpers/helper';
+import './AddContactForm.css';
+
+class AddContactForm extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      firstName: '',
+      lastName: '',
+      imageUrl: '',
+      firstNameError: false,
+      lastNameError: false,
+      imageUrlError: false
+    };
+  }
+
+  onFormInputChange = (event, type) => {
+    const { target } = event;
+    const newInput = target.value.trim();
+    this.setState({
+      [type]: newInput,
+      [`${type}Error`]: !validateInput(newInput, GET_REGEX[type])
```

Comment:
Now that you updated the GET_REGEX constant, you can even get rid of the type param. Just add a `name="firstName"` property on the input and in `onFormInputChange` function use target.name to get that property.